examples/torque_driven_ocp/phase_transition_uneven_variable_number.py

```diff
@@ -112,7 +112,8 @@ def main():
     # --- Prepare the ocp --- #
     ocp = prepare_ocp()
 
-    sol = ocp.solve()
+    show_options = dict(show_bounds=True)
+    sol = ocp.solve(show_online_optim=True, show_options=show_options)
```

Comment:
can you replace this by the much clearer : 
`sol = ocp.solve(show_online_optim=True, show_options={show_bounds=True})